__tests__/app.test.js

```diff
@@ -24,10 +24,24 @@ describe('/api',()=>{
                         expect(topic).toHaveProperty('slug');
                         expect(topic).toHaveProperty('description')
                     })
+                    expect(topics).toHaveLength(3)
                 })
         })
+        
+    })
+    describe('/api endpoints',()=>{
+        test('Should respond with all the endpoints',()=>{
+            return request(app).get('/api').expect(200).then(({body})=>{
+                const {endpoints} = body;
+                expect(endpoints).toHaveProperty('GET /api')
+                expect(endpoints).toHaveProperty('GET /api/topics')
```

Comment:
One thing you could do to avoid coming back and updating this test every time you add a new endpoint... You can require the endpoints file in and assert that it matches the file content 👍